docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_gitops_basic/_index.md

```diff
@@ -193,7 +193,7 @@ With Namespace-level GitOps config, the goal is to have Kubernetes resources dep
   
     ![kubectl get pods -n hello-arc -w](./13.png)
 
-  - **Your fork** of the "Azure Arc Jumpstart" repository. Open the [*hello_arc.yaml*](https://github.com/zaidmohd/azure_arc/tree/main/artifacts/hello-arc/yaml/hello_arc.yaml) file.
+  - In **Your fork** of the "Azure Arc Jumpstart" repository, open the *hello_arc.yaml* file (/azure_arc/tree/main/artifacts/hello-arc/yaml/hello_arc.yaml).
```

Comment:
Would tree actually be in the real path on a user's fork on their local machine? I think "tree" needs to come out of the path here along with the main folder. 
![image](https://user-images.githubusercontent.com/32211549/149419542-45194f7b-26c4-4d9c-a8a0-481facebbcf1.png)